ui/ui-frontend/projects/pastis/src/app/core/services/archival-profile-unit-api.service.ts

```diff
@@ -67,7 +67,7 @@ export class ArchivalProfileUnitApiService extends BaseHttpClient<ArchivalProfil
   }
 
   updateProfilePua(archivalUnitProfile: ArchivalProfileUnit, headers?: HttpHeaders): Observable<ArchivalProfileUnit> {
-    return this.http.put<ArchivalProfileUnit>(this.apiUrl + archivalUnitProfile.identifier, archivalUnitProfile, { headers });
+    return this.http.put<ArchivalProfileUnit>(this.apiUrl + archivalUnitProfile.id, archivalUnitProfile, { headers });
```

Comment:
As-tu vérifier si on a pas d'autre cas (puisque c'est déjà le second) ?
Je pense notamment à :
- ProfileService.updateProfileFilePa
- ProfilePreviewComponent.downloadProfile

Reply:
Je vais merger cette PR, mais je vais laisser le ticket ouvert pour continuer d'investiguer.

Reply:
J'ai vérifié les différents appels (ceux que tu mentionnes et d'autres) et ça m'a l'air bon. Je vais passer le ticket en "In testing".